common/source/docs/common-uavcan-setup-advanced.rst

```diff
@@ -61,7 +61,7 @@ Each UAVCAN ESC or Servo will have an programmed ID or channel address correspon
 There are three parameters that determine which autopilot servo/motor channels are sent to the CAN ESC and/or Servos:
 For the examples below, the values are shown for CAN driver #1.
 
--  :ref:`CAN_D1_UC_NODE<CAN_D1_UC_NODE>` - which is the node ID of the autopilot sending the commands to the ESCs so that there can be differentiation between multiple sources on the CAN bus
+-  :ref:`CAN_D1_UC_NODE<CAN_D1_UC_NODE>` - which is the node ID of the autopilot sending the commands to the ESCs so that there can be differentiation between multiple sources on the CAN bus. When a device with this ID is attached and recognized, it is entered into a database which is stored between power cycles. If multiple devices with the same ID and different hardware IDs (swapping smart batteries, for example, without cycling power) are used, this will require the use of the :ref:`CAN_D1_UC_OPTION<CAN_D1_UC_OPTION>` parameter to allow the database to be reset on the next boot or conflicts in the database to be ignored.
```

Comment:
The CAN_D1_UC_NODE parameter is for the CAN node ID of ArduPilot. The DNA database (dynamic node allocation database) holds node IDs of other nodes, such as all the peripherals. The database stores the node ID and the corresponding hardware identifier.
The UC_OPTION parameter gives methods to either clear the database, or ignore conflicts. A conflict occurs when the same node ID announces itself on the bus with different hardware identifiers.

Reply:
think its fixed now...